bundler.gemspec

```diff
@@ -43,7 +44,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency "ronn",       "~> 0.7.3"
   s.add_development_dependency "rspec",      "~> 3.6"
 
-  s.files = `git ls-files -z`.split("\x0").select {|f| f.match(%r{^(lib|exe)/}) }
+  gemspec_path = Pathname.glob("**/bundler.gemspec").first.dirname.to_path
+  s.files = `git -C #{gemspec_path} ls-files -z`.split("\x0").select {|f| f.match(%r{^(lib|exe)/}) }
```

Comment:
There's no need in `Pathname` IMO. What you need is to figure our dirname of current `__FILE__`. So you can either use `File.dirname(__FILE__)` here, or simply using `__dir__`. Also it worth to shell-escape this path otherwise it will break on most of Windows setups (where spaces in paths are very common):

``` ruby
s.files = `git -C #{Shellwords.escape __dir__} ls-files -z`.split("\x0").select {|f| f.match(%r{^(lib|exe)/}) }
```